### PR TITLE
[yaml] various inline provider doc fixes

### PR DESCRIPTION
**Pull request by Polber**
Fixing a couple formatting issues with various providers.

- add args to Combine doc
- StripErrorMetadata code was not in code block
- StripErrorMetadata ignore super class (PTransform) args (label)

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by Polber** [comment]
R: @robertwb

File: sdks/python/apache_beam/yaml/yaml_combine.py [hunk 1]
```diff
@@ -94,6 +94,12 @@ class PyJsYamlCombine(beam.PTransform):
 
   See also the documentation on
   [YAML Aggregation](https://beam.apache.org/documentation/sdks/yaml-combine/).
+
+  Args:
+    group_by: The field to aggregate on.
```
Comment (mravi) [inline sdks/python/apache_beam/yaml/yaml_combine.py:99]: super nit: `field(s)`

File: sdks/python/apache_beam/yaml/yaml_mapping.py [hunk 2]
```diff
@@ -453,6 +453,9 @@ class _StripErrorMetadata(beam.PTransform):
 
   _ERROR_FIELD_NAMES = ('failed_row', 'element', 'record')
 
+  def __init__(self):
+    super().__init__(label=None)
```
Comment (robertwb) [inline sdks/python/apache_beam/yaml/yaml_mapping.py:457]: Why was this needed?
